workorders.lic

```diff
@@ -472,13 +477,20 @@ class WorkOrders
         stow_tool(right_hand)
         bput("Mark my #{recipe['noun']} at 5", 'You measure')
         bput("Break my #{recipe['noun']}", 'You carefully')
-        bput('stow left', 'You put')
+        /(\d+)/ =~ bput("count my first #{recipe['noun']}", 'You count out \d+ uses remaining.')
+        if Regexp.last_match(1).to_i == 5
+          bput("stow my second #{recipe['noun']}", "You put")
+        else
+          bput("stow my first #{recipe['noun']}", "You put")
+        end
         bundle_item(recipe['noun'], info['logbook'])
+        dispose(recipe['noun'])
```

Comment:
How does this work?

If you stow one of the items, and then bundle the other, when would you ever have something to dispose?